plugins/woocommerce-admin/client/products/fields/variations/variations.tsx

```diff
@@ -98,12 +132,91 @@ export const Variations: React.FC = () => {
 								/>
 							) ) }
 						</div>
-						<div className="woocommerce-product-variations__price">
+						<div
+							className={ classNames(
+								'woocommerce-product-variations__price',
+								{
+									'woocommerce-product-variations__price--fade':
+										variation.status === 'private',
+								}
+							) }
+						>
 							{ formatAmount( variation.price ) }
 						</div>
-						<div className="woocommerce-product-variations__quantity">
+						<div
+							className={ classNames(
+								'woocommerce-product-variations__quantity',
+								{
+									'woocommerce-product-variations__quantity--fade':
+										variation.status === 'private',
+								}
+							) }
+						>
 							{ getProductStockStatus( variation ) }
 						</div>
+						<div className="woocommerce-product-variations__actions">
+							{ variation.status === 'private' && (
+								<Tooltip
+									position="top center"
+									text={ NOT_VISIBLE_TEXT }
+								>
+									<Button
+										className="components-button--hidden"
+										aria-label={
+											isUpdating[ variation.id ]
+												? UPDATING_TEXT
+												: NOT_VISIBLE_TEXT
+										}
+										aria-disabled={
```

Comment:
Because of the disabled styling, it's a little hard to see the spinner or tooltips when this occurs.  Could we move the spinner out of the button?


<img width="106" alt="Screen Shot 2022-12-15 at 2 43 23 PM" src="https://user-images.githubusercontent.com/10561050/207983466-f6def800-616c-40d0-8fd9-2bc93acc54e5.png">

If we move the updating logic above, we can avoid some of the condition checks inside both of these buttons:

```
{ isUpdating[ variation.id ] ? ( <Spinner /> ) }
{ ! isUpdating[ variation.id ] && variation.status === 'private' && ... }
{ ! isUpdating[ variation.id ] && variation.status === 'publish' && ... }
```

Reply:
The spinner opacity was fixed here -> [6c6f25c](https://github.com/woocommerce/woocommerce/pull/36020/commits/6c6f25cf21cf3c4969d1d2860645531f54ca70af)

Reply:
Ah that makes total sense on the a11y concerns.  Thanks for the styling fix up!